src/cnglob.h

```diff
@@ -271,6 +271,7 @@ constexpr double tAbs0F						// 0 F in Rankine
 #else
 	= 459.67;
 #endif
+constexpr double tol_tF = 1.e-12; // temperature tolerance (degF)
 
```

Comment:
The tolerance used in these two cases serves to prevent division by zero; it does not correspond to a precision on the fitted value.  (The fitting tolerance and number of iterations within the secant method have not been altered.) Even with the tighter tolerance, the `secant' method will likely fail if the if the condition  in these functions is met. (It seems obvious that the the two functions are related, but if there is some justification for using a different value in one or the other, please specify). Given that there are at least two instances of solver functions that return AMFs and contain terms with degF temperature differences  in their divisors, the definition of a standard tolerance within some broader scope seems merited.